src/datasources/cache/cache.router.ts

```diff
@@ -14,6 +14,7 @@ export class CacheRouter {
   private static readonly MASTER_COPIES_KEY = 'master_copies';
   private static readonly MESSAGE_KEY = 'message';
   private static readonly MESSAGES_KEY = 'messages';
+  private static readonly MODULES_KEY = 'modules';
```

Comment:
Nitpick: if I'm not wrong we are storing Safe addresses behind this key, so I'd use another name, like `module_safes` (taken from `owner_safes` example).

Reply:
I changed it in 970f68198f654722658cdfce7929e210556aecb3 / bba9179e176321e4aaf6eacd0b7a4c475ded8417 to `module_safes`.